README.md

```diff
@@ -157,7 +157,7 @@ Snowflake has been tested with Ubuntu server, CentOS, RHEL, OpenSUSE, FreeBSD, O
 
 
 
-<h2>Documentations:</h2>
+<h2>Documentation:</h2>
 https://github.com/subhra74/snowflake/wiki
```

Comment:
Can you also convert this line into a hyperlink?
```
[https://github.com/subhra74/snowflake/wiki](https://github.com/subhra74/snowflake/wiki)
```

Reply:
Good catch. It's done.